### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@pixdrift @TomSweeneyRedHat @umohnani8 PTAL

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,287 @@
...
+    Block use of /proc/acpi and /proc/keys from inside containers
+    Fix handling of --registries-conf
+    Fix becoming a maintainer link
+    add optional CI test fo darwin
+    Don't pass a nil error to errors.Wrapf()
+    image filter test: use kubernetes/pause as a "since"
+    Add --cidfile option to from
+    vendor: update containers/storage
+    Contributors need to find the CONTRIBUTOR.md file easier
+    Add a --loglevel option to build-with-dockerfile
```
Comment (umohnani8) [inline CHANGELOG.md]: Should this be build-using-dockerfile instead?

**Comment by umohnani8** [comment]
One question, otherwise LGTM.

File: CHANGELOG.md (hunk 1 above)
Comment (rhatdan) [reply CHANGELOG.md]: Well these are the actual changelogs.

**Comment by rhatdan** [comment]
@rh-atomic-bot r=TomSweeneyRedHat

**Pull request closed without merging**